### DISCUSSION TITLE
iOS15已有pro账号，下载的免费版本如何切换为pro版本呢

### DISCUSSION THREAD

**Discussion by guorui22** [getlantern/lantern-forum-cn#531] [Q&A]
![IMG_4764](https://github.com/getlantern/lantern-forum-cn/assets/8207719/d1365396-4e19-422b-9e3c-d967c47e729f)

**Comment by guorui22** [comment]
反馈问题也显示错误

**Reply by MaxMKMax** [reply]
@guorui22 您好. 谢谢您提供的信息. 我帮您反馈一下.
